# HG changeset patch
# User Markus Mützel <markus.muetzel@gmx.de>
# Date 1711650847 -3600
#      Thu Mar 28 19:34:07 2024 +0100
# Node ID b8ffc513a4400c87a755369c358955ce3a868d9c
# Parent  29a9f1e01a610135c45de7236a5987069055c234
Include <cassert> in file that uses "assert".

* src/remez.cc: The header <cassert> happened to be include via some headers
from Octave before. That is no longer the case for Octave 10. Include the
header directly in a file that requires it.

diff -r 29a9f1e01a61 -r b8ffc513a440 src/remez.cc
--- a/src/remez.cc	Tue Feb 06 11:50:00 2024 -0500
+++ b/src/remez.cc	Thu Mar 28 19:34:07 2024 +0100
@@ -27,6 +27,7 @@
  *************************************************************************/
 
 #include <octave/oct.h>
+#include <cassert>
 #include <cmath>
 
 #ifndef OCTAVE_LOCAL_BUFFER
